diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index 83a46645ee08815fe8f41da81d0a753480f8192e..54a17b5bbbbe082737d7e542e10949b8bee31d6a 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -106,6 +106,7 @@ class ProfileSyncService : public SyncService,
   // Initializes the object. This must be called at most once, and
   // immediately after an object of this class is constructed.
   // TODO(mastiz): Rename this to Start().
+  virtual
   void Initialize();
 
   // SyncService implementation
@@ -254,6 +255,7 @@ class ProfileSyncService : public SyncService,
 
   static std::string GenerateCacheGUIDForTest();
 
+ BRAVE_PROFILE_SYNC_SERVICE_H_
  private:
   // Passed as an argument to StopImpl to control whether or not the sync
   // engine should clear its data when it shuts down. See StopImpl for more
